includes/gpio.h

```diff
@@ -0,0 +1,84 @@
+#ifndef GPIO_H
+#define GPIO_H
+
+#include <stdint.h>
+
+typedef struct 	GPIO_config_s
```

Comment:
Do not use tabs for indentation. This may look weird for other people (and in github reviews). Use spaces instead.